comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/dataset/DatasetExample.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.research.commentupdater.dataset
+
+import org.jetbrains.research.commentupdater.models.MethodMetric
+
+data class DatasetExample(
+    val oldCode: String,
+    val newCode: String,
+    val oldComment: String,
+    val newComment: String,
+    val oldMethodName: String,
+    val newMethodName: String,
+    val commitId: String,
+    val commitTime: String,
+    val fileName: String,
```

Comment:
what does the plugin store here if the method was moved to another file?

Reply:
Plugin saves a name of a new file version. (changed this variable name to `newFileName`)